packages/block-directory/src/components/downloadable-block-info/style.scss

```diff
@@ -0,0 +1,24 @@
+.block-directory-downloadable-block-info__content {
+	font-size: 14px;
+}
+
+.block-directory-downloadable-block-info__row {
+	display: flex;
+	justify-content: space-between;
+	color: $dark-gray-400;
+	margin-top: 8px;
+
+	.block-directory-downloadable-block-info__column {
+		display: flex;
+		align-items: flex-start;
+
+		.dashicon {
+			font-size: 16px;
+			margin-right: 4px;
+		}
+
+		.dashicons-chart-line {
+			font-weight: 900;
```

Comment:
The font weight doesn't appear to do anything on the dashicons icon font because it only has one weight. So it should be possible to remove this rule?

Reply:
Removed 👍